web/js/panel.js

```diff
@@ -2048,11 +2048,11 @@ function $drawTrackSegment($widget) {
     var $ep1, $ep2;
     [$ep1, $ep2] = $getEndPoints$($widget);
     if (typeof $ep1 === "undefined") {
-    		jmri.log("ERROR: can't draw tracksegment " + $widget.ident + ": connect1: " + $widget.connect1name + "." + $widget.type1 + " undefined.");
+    		console.error("ERROR: can't draw tracksegment " + $widget.ident + ": connect1: " + $widget.connect1name + "." + $widget.type1 + " undefined.");
```

Comment:
Note: You *cannot* rely on the console existing--there is no reason not to expect `console.error` to crash out the JavaScript completely on user's devices, so please revert this, or come up with a reliable alternative.

There is *zero* benefit to writing code to making unit testing easy if if crashes out entirely for users.